Gopkg.toml

```diff
@@ -104,3 +104,22 @@ required = [
 [[constraint]]
   name = "github.com/apache/camel-k"
   version = "0.3.4"
+  
+
+# The dependencies below are required for opencensus.
```

Comment:
Can this be "The next 4 dependencies" so if someone appends a new one it won't get added to the list? Or copy this comment to each override just to be safe.